evennia/utils/evmenu.py

```diff
@@ -64,10 +68,37 @@ def another_node(caller, input_string, **kwargs):
 The return values must be given in the above order, but each can be
 returned as None as well. If the options are returned as None, the
 menu is immediately exited and the default "look" command is called.
-
-    text (str, tuple or None): Text shown at this node. If a tuple, the
-        second element in the tuple is a help text to display at this
-        node when the user enters the menu help command there.
+    text (str, tuple, dict or None): Text shown at this node. If a dict
+        it can also contain various formatting instructions. In such a 
+        case the followng keys are already supported:        
+            - 'text' (str): The text of the node.
+            - 'help' (str): The help text of the node.
+            - 'footer' (str): Footer text for the node.
+            - 'format' (str): String dictating the layout of the node if
+                the default layout is not being used. It currently
+                supports two options:
+                    - 'suppress': The node is not drawn at all
+                    - 'scroll': The node is framed inside the ANSI image
+                        of a scroll.
```

Comment:
Such a specialized formatting option, no matter how cool-looking, does not belong in core. This is something that will need to be supplied via external means.

Reply:
I chose to use this function primarily because I simply had it about. 

It occurs to me that I could very easily replace this with a 'no_bars' format (being able to suppress the bars on certain nodes being one of the original motivations of the project). I'll swap out scroll for no_bars unless you have some other alternate format you would like to use.

Reply:
I think it would be better to make this more flexible; I elaborate on this below.

Reply:
I think I have addressed all of the issues you elaborated on below.